{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -172,24 +172,25 @@ def test_mint(token, owner, receiver):
     Create an approved amount of tokens.
     """
     totalSupply = token.totalSupply()
-    assert totalSupply == 1000
+    assert totalSupply == {{cookiecutter.premint_amount}}
```

Comment:
hmm, might be better to do something like this instead
```suggestion
    PREMINT_AMOUNT = {{cookiecutter.premint_amount}}
    assert totalSupply == PREMINT_AMOUNT
```

Reply:
Should also probably just make PREMINT_AMOUNT a constant at the top of this file